addons/mail/models/mail_mail.py

```diff
@@ -196,7 +196,11 @@ def _send_prepare_values(self, partner=None):
         body = self._send_prepare_body()
         body_alternative = tools.html2plaintext(body)
         if partner:
-            email_to = [tools.formataddr((partner.name or 'False', partner.email or 'False'))]
+            emails_normalized = partner._compute_email_to(single=False, format_addr=False)[0]
+            email_to = [
+                tools.formataddr((partner.name or u"False", email or u"False"))
+                for email in emails_normalized.split(',')
+            ]
```

Comment:
You cannot use `split(',')` to split multi recipients, it doesn't work for `"Doe, Jhon" john.doe@example.com`

Reply:
Returned emails are normalized, aka containing only the email part. But actually I changed the tool method to avoid manipulating strings.